x-pack/plugins/maps/public/shared/layers/sources/wms_source.js

```diff
@@ -13,12 +13,18 @@ import {
 
 import { AbstractTMSSource } from './tms_source';
 import { TileLayer } from '../tile_layer';
+import { i18n } from '@kbn/i18n';
+import { getDataSourceLabel, getUrlLabel } from '../../../../common/i18n_getters';
 
 export class WMSSource extends AbstractTMSSource {
 
   static type = 'WMS';
-  static title = 'Web Map Service';
-  static description = 'Maps from OGC Standard WMS';
+  static title = i18n.translate('xpack.maps.source.wms.title', {
+    defaultMessage: 'Web Map Service'
+  });
+  static description = i18n.translate('xpack.maps.source.wms.description', {
```

Comment:
I'm not sure how this is consumed, but i guess this might be adhering more to the guidelines.

```suggestion
  static description = i18n.translate('xpack.maps.source.wmsDescription', {
```